llvm/test/CodeGen/DirectX/ShaderFlags/iswave.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: opt -S --passes="print-dx-shader-flags" 2>&1 %s | FileCheck %s
+
+target triple = "dxil-pc-shadermodel6.7-library"
+
+; CHECK: ; Shader Flags Value: 0x00080000
+; CHECK: ; Note: shader requires additional functionality:
+; CHECK-NEXT: ;       Wave level operations
+; CHECK-NEXT: ; Note: extra DXIL module flags:
+; CHECK-NEXT: {{^;$}}
+
+; TODO: is there a way that we can split the input file into many?
+; as the shader flag will be true if any are set
```

Comment:
The other option, that I think would be best, is to add this `RUN` line to each of the operation tests, eg in `WaveReadLaneAt.ll` in the parent directory.